packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -170,7 +171,10 @@ export function getAnnotationData(
                 hotspotData &&
                 hotspotData.result &&
                 hotspotData.status === 'complete'
-                    ? isRecurrentHotspot(mutation, hotspotData.result)
+                    ? // if mutation type is splice, get splice hotspot, otherwise get recurrent hotspot
+                      mutation.mutationType?.toLowerCase().includes('splice')
+                        ? isSpliceHotspot(mutation, hotspotData.result)
+                        : isRecurrentHotspot(mutation, hotspotData.result)
```

Comment:
Could you confirm the logic here? If the mutation type is `splice`, then check if the hotspots data has `splice` type, for all other types of mutations check if it has recurrent hotspots(`single` and `indel`)

Reply:
Should we just change the `isRecurrentHotspot` logic and include splice there instead? That would simplify the logic. Also I think you can simply do this: 
```suggestion
                    ? isSpliceHotspot(mutation, hotspotData.result) || isRecurrentHotspot(mutation, hotspotData.result)
```

Reply:
as we discussed, only splice mutations should match splice hotspots

Reply:
Makes sense.

Can we avoid using nested ternary operators though? They are not very easy to read, and quite easy to mess up.